frontend/components/forms/fields/DropdownWrapper/DropdownWrapper.tsx

```diff
@@ -260,7 +280,6 @@ const DropdownWrapper = ({
         color: COLORS["ui-fleet-black-50"],
         fontStyle: "italic",
         cursor: "not-allowed",
-        pointerEvents: "none",
```

Comment:
Why were we able to drop this?

Reply:
To view, you can run `yarn storybook` and check out `localhost:6006`, Commit incoming to fix the issue of a dropdown's options/tooltips being cut off in Storybook's canvas viewport.